python/lsst/ctrl/mpexec/singleQuantumExecutor.py

```diff
@@ -78,17 +78,15 @@ class SingleQuantumExecutor(QuantumExecutor):
         instead.
     taskFactory : `~lsst.pipe.base.TaskFactory`
         Instance of a task factory.
-    skipExistingIn : `list` [ `str` ], optional
-        Accepts list of collections, if all Quantum outputs already exist in
-        the specified list of collections then that Quantum will not be rerun.
-        If `None` then butler output RUN collection is searched for existing
-        datasets. If empty list then there no check for existing outputs (which
-        could result in conflicts when datasets are stored).
+    extendRun : bool, optional
+        True if execution is extending an existing run instead of creating a
+        new empty run. Quanta with complete outputs will be skipped. Partial
+        outputs from failed quanta will be removed if ``clobberOutputs`` is
+        `True`, otherwise an exception will be raised.
```

Comment:
What is the behavior if `extendRun` is false?

Reply:
If `extendRun` is not set then it will just try to write to the output run without any checks, which should fail if the same dataset is already there.

Reply:
Can you say that explicitly?